src/app/modules/convai-checker/perspective-status.component.ts

```diff
@@ -500,24 +499,24 @@ export class PerspectiveStatusComponent implements OnChanges, OnInit, AfterViewI
     }
   }
 
-  private getShouldHideStatusWidget(loadStart: boolean): boolean {
+  private getShouldHideStatusWidget(): boolean {
     let shouldHide = false;
 
     if (!this.showFeedbackForLowScores) {
-      shouldHide = shouldHide || loadStart || this.score < this.neutralScoreThreshold;
+      shouldHide = shouldHide || this.score < this.neutralScoreThreshold;
     }
 
     if (!this.showFeedbackForNeutralScores) {
-      shouldHide = shouldHide || loadStart || (
+      shouldHide = shouldHide || (
         this.score >= this.neutralScoreThreshold
         && this.score < this.toxicScoreThreshold);
     }
 
     return shouldHide;
   }
 
-  private getUpdateStatusWidgetVisibilityAnimation(loadStart: boolean): TimelineMax {
-    const hide = this.getShouldHideStatusWidget(loadStart);
+  private getUpdateStatusWidgetVisibilityAnimation(): TimelineMax {
+    const hide = this.getShouldHideStatusWidget();
```

Comment:
Should this variable be used below? Or should we just remove it?

Reply:
It is used; see line 533.

Reply:
Oops, GitHub hid those lines.